filters/GridDecimationFilter.cpp

```diff
@@ -107,9 +111,9 @@ void GridDecimationFilter::processOne(BOX2D bounds, PointRef& point, PointViewPt
 
 void GridDecimationFilter::createGrid(BOX2D bounds)
 {
-    double d_width = std::floor((bounds.maxx - bounds.minx) / m_args->m_edgeLength) + 1;
-    double d_height = std::floor((bounds.maxy - bounds.miny) / m_args->m_edgeLength) + 1;
-    
+    double d_width = std::round((bounds.maxx - bounds.minx) / m_args->m_edgeLength);
```

Comment:
I think the original is correct here. Consider an edge length of 1 and a min/max of [0, 1.1].

Reply:
I will add that if you have, say, an edge length of 10 and bounds of [0, 30], you need 4 cells, as the boundaries are:
[0, 10), [10, 20), [20, 30), [30, 40).